src/test/scala/firrtlTests/FlattenTests.scala

```diff
@@ -8,7 +8,7 @@ import org.scalatest.junit.JUnitRunner
 import firrtl.ir.Circuit
 import firrtl.Parser
 import firrtl.passes.PassExceptions
-import firrtl.annotations.{Annotation, CircuitName, ComponentName, ModuleName, Named}
+import firrtl.annotations.{Annotation, CircuitName, ComponentName, ModuleName}
```

Comment:
Seems like only imports at lines 11,12 here are actually used. All others are unused. But maybe this should
be saved for a style PR